rust/xaynet-server/src/storage/impls.rs

```diff
@@ -5,7 +5,7 @@ use redis::{ErrorKind, FromRedisValue, RedisError, RedisResult, RedisWrite, ToRe
 use thiserror::Error;
 use xaynet_core::{
     crypto::{ByteObject, PublicEncryptKey, PublicSigningKey},
-    mask::{EncryptedMaskSeed, MaskMany},
+    mask::{EncryptedMaskSeed, MaskObject},
```

Comment:
Looks like a rebasing issue?

Reply:
rebasing issue because it was previously `MaskMany` or because is now `MaskObject`?